### ISSUE TITLE
Properly compute standard property for nested SCSS properties so that rules such as experimental-property-usage works fine

### ISSUE THREAD

**Issue by hcomnetworkers** [racodond/sonar-css-plugin#6]
Hi,

I get the violation '_Remove this usage of the experimental "size" property._' on a file but it seems to be wrong in this case.

Here is the rule in my .scss file:
```
@mixin font-normal {
  color: $color-font;
  font: {
    family: Arial, Helvetica, sans-serif;
    size: $fontSize-m;
    weight: 300;
  }
  line-height: $lineHeight-m;
}
```

The violation is right in the row containing the `size`.

It seems that the outer `font` is not detected here but it is valid CSS.
It is quite interesting that the other rules `family` and `weight` got no violation :-)

Is it possible to adjust the rule?

Thanks in advance!

**Comment by racodond** [comment]
Thanks for your feedback! I'll get back to you when I get back home beginning of next week.
